primefaces/src/main/java/org/primefaces/component/datatable/DataTable.java

```diff
@@ -61,8 +61,9 @@
 @ResourceDependency(library = "primefaces", name = "jquery/jquery.js")
 @ResourceDependency(library = "primefaces", name = "jquery/jquery-plugins.js")
 @ResourceDependency(library = "primefaces", name = "core.js")
-@ResourceDependency(library = "primefaces", name = "components.js")
 @ResourceDependency(library = "primefaces", name = "touch/touchswipe.js")
+@ResourceDependency(library = "primefaces", name = "datatable/datatable.js")
```

Comment:
why making datatable a own file? one of the most used components

Reply:
Only because its 200K so it reduces components.js by 100K so we do have pages that don't use DataTables so its loading that extra for no reason?  I think once a component is over 100K it should always be in its own file?

Reply:
the idea was to bundle most common components - ignoring size or everything else
i would not split it up

Reply:
OK Datatable changes reverted.